test/api/v3/unit/models/group.test.js

```diff
@@ -1173,11 +1173,12 @@ describe('Group Model', () => {
         it('updates participating members with rewards', async () => {
           await party.finishQuest(quest);
 
-          expect(User.update).to.be.calledOnce;
```

Comment:
There's still no test for the retry logic. You can use the sinon spy library to provide the results you need for failures to update.